Symfony/CS/Fixer.php

```diff
@@ -311,6 +312,48 @@ public function fixFile(\SplFileInfo $file, array $fixers, $dryRun, $diff, FileC
         return $fixInfo;
     }
 
+    /**
+     * @param \SplFileInfo     $file
+     * @param Tokens           $tokens
+     * @param FixerInterface[] $fixers
+     *
+     * @return string[]
+     */
+    public function fixFileTokens(\SplFileInfo $file, Tokens $tokens, array $fixers)
```

Comment:
A alternative would be to mock the Fixer in the test and up the visibility of the method there,
would that be more acceptable?